packages/mask/content-script/components/InjectedComponents/SetupGuide/VerifyNextID.tsx

```diff
@@ -189,7 +189,7 @@ export function VerifyNextID({ onClose }: VerifyNextIDProps) {
             await handleVerifyNextID(personaInfo, userId)
             Telemetry.captureEvent(EventType.Access, EventID.EntryPopupSocialAccountVerifyTwitter)
         }
-        queryClient.removeQueries({
```

Comment:
why change this? @nuanyang233

Reply:
Other components use the `isPending` status of this query. Changing to `removeQueries` will cause some bugs.